addon/components/-private/animated-popper.js

```diff
@@ -119,12 +121,18 @@ export default class AnimatedPopperComponent extends Component {
     super(...arguments);
 
     this.isOpenDidChange();
+
+    run.schedule('render', () => {
```

Comment:
I think this should be `afterRender`. Scheduling into `render` doesn't deterministically result in this code running after the rendering has happened, rendering itself could already have been, or could have not been, scheduled.

Reply:
Yeah, in general I'm surprised to see so much code here using the `render` queue. If the goal is merely to run "after the current stack/runloop-frame has flushed" then `actions` would be correct.

Reply:
`afterRender` was my initial thought but I ran into some problems that made me wonder if 


`render` would not be better (I think it was because by the time `afterRender` callbacks were called the popper was already initialized with a different target).
I think that by digging more into `ember-popper` and `popper.js` I fixed the problem but forgot to use `afterRender` again. I put it back 👍